content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,199 @@
+|:-----------|:--------|:------------------------------------------------------|
+| `market`   | String  | Returns trades executed in a particular market in which XRP is either the base or counter currency. Use a suggested format of `<basecurrency>_<countercurrency>` to provide valid values like `xrp_btc`, `btc_xrp`, `xrp_usd`, or `usd_xrp`, for example. |
+| `last_tid` | Integer | _(Optional)_ Returns trades executed after a specific trade ID. For example, XRP Charts can use this parameter to get all trades after the last set of trades recorded in site data, ensuring that it has recorded all trades. |
+| `limit`    | Integer | _(Optional)_ Returns no more than a specific number of trades in a response. |
+
+
+### Response Format
+
+A successful response must be a JSON array of objects, one for each trade. Use of these exact field names is optional. While having access to all of these fields would be useful, your endpoint isn't required to provide the ones marked as _(Optional)_.
+
+***TODO: `tid` should be required, is that right? Or is there some other way to determine uniqueness?***
+
+| Field       | Type             | Description                                 |
+|:------------|:-----------------|:--------------------------------------------|
+| `price`     | String or Number | Exchange rate of the trade.                 |
+| `amount`    | String or Number | Amount of XRP bought or sold.               |
+| `timestamp` | String           | Time at which the trade was executed in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) or [Unix time](https://en.wikipedia.org/wiki/Unix_time) format. |
+| `tid`       | Integer          | _(Optional)_ Unique identifier of the trade. Ideally, make this a sequential integer. |
+| `type`      | String           | _(Optional)_ Type of trade. For example, valid values can include `buy` and `sell`. |
+| `size`      | String or Number | _(Optional)_ Amount of counter currency traded. |
+| `count`     | Integer          | _(Optional)_ Number of trade objects returned in the response. |
+
+
+### Examples
+
+***TODO: examples below okay - esp use of `limit` in URL and `count` in response message?***
+
+#### Example Request
+
+```
+GET https://api.example.com/v1/trades?market=xrp_btc&last_tid=75208825&limit=500
+```
+
+#### Example Response
+
+```json
+[
+   {
+      "tid":75209326,
+      "type":"buy",
+      "price":"0.57201",
+      "amount":"4954.0744",
+      "size":"2833.7801",
+      "timestamp":"2018-10-01T12:35:11.000Z"
+   },
+   ...
+   {
+      "tid":75208826,
+      "type":"sell",
+      "price":"0.57201",
+      "amount":"4954.0744",
+      "size":"2833.7801",
+      "timestamp":"2018-10-01T12:31:16.000Z"
+   }
+   "count":"500"
+]
+```
+
+
+
+## Provide a Get Current Order Book Endpoint
+
+Provide a REST API endpoint that returns data about the current order book in a particular market.
+
+XRP Charts will query this endpoint about every 30 seconds.
```

Comment:
Does this frequency get adjusted depending on the data Charts gets back?

Reply:
Great question - I will ask!